registry/api/v2/descriptors.go

```diff
@@ -1595,6 +1595,133 @@ var routeDescriptors = []RouteDescriptor{
 			},
 		},
 	},
+	{
+		Name:        RouteNameReferrers,
+		Path:        "/v2/{name:" + reference.NameRegexp.String() + "}/referrers/{digest:" + digest.DigestRegexp.String() + "}",
+		Entity:      "Referrers",
+		Description: `Retrieve information about referrers.`,
+		Methods: []MethodDescriptor{
+			{
+				Method:      "GET",
+				Description: "Fetch the referrers of the artifact identified by `digest`.",
+				Requests: []RequestDescriptor{
+					{
+						Name:        "referrers",
+						Description: "Request an unabridged list of referrers.",
+						Successes: []ResponseDescriptor{
+							{
+								Description: "Returns an image index containing all referrers as a json response.",
+								StatusCode:  http.StatusOK,
+								Headers: []ParameterDescriptor{
+									{
+										Name:        "Content-Length",
+										Type:        "integer",
+										Description: "Length of the JSON response body.",
+										Format:      "<length>",
+									},
+									linkHeader,
+								},
+								Body: BodyDescriptor{
+									ContentType: "application/json",
```

Comment:
Should it be `application/vnd.oci.image.index.v1+json`?

Reply:
replaced.